test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -134,16 +135,20 @@ void dev::test::RandomCode::parseTestWithTypes(std::string& _test, std::map<std:
 				replace = test::RandomCode::randomUniIntHex(dev::u256("100000"), dev::u256("36028797018963967"));
 			else if (type == "[DESTADDRESS]")
 			{
-				Address address = _options.getRandomAddress(RandomCodeOptions::AddressType::PrecompiledOrStateOrCreate);
+				Address address = _options.getRandomAddress(RandomCodeOptions::AddressType::DestinationAccount);
 				if (address != ZeroAddress) //else transaction creation
 					replace = "0x" + toString(address);
 			}
-			else if (type == "[ADDRESS]")
-				replace = toString(_options.getRandomAddress(RandomCodeOptions::AddressType::StateAccount));
-			else if (type == "[0xADDRESS]")
+			else if (type == "[ADDRESS]") {
+				Address destAddress = _options.getRandomAddress(RandomCodeOptions::AddressType::StateAccount);
+				_options.addAddress(destAddress, RandomCodeOptions::AddressType::DestinationAccount);
+				replace = toString(destAddress);
+			}
+			else if (type == "[0xADDRESS]") {
 				replace = "0x" + toString(_options.getRandomAddress(RandomCodeOptions::AddressType::StateAccount));
+			}
```

Comment:
Spurious `{` and `}`.  Read https://github.com/ethereum/cpp-ethereum/blob/develop/CodingStandards.txt